xla/service/cpu/onednn_util.h

```diff
@@ -59,7 +90,7 @@ dnnl::stream MakeOneDnnStream(
     const dnnl::engine& cpu_engine,
     dnnl::threadpool_interop::threadpool_iface* thread_pool);
 
-typedef BackendConfig::BackendConfigOneofCase BackendConfigOneofCase;
+typedef BackendConfig::BackendConfigOneofCase BackendConfigOneofCase; 
```

Comment:
Please remove `typedef` and use the full type in header file.

Reply:
Fixed.